tests/cli/compile_test.py

```diff
@@ -309,6 +309,54 @@ async def test_index_nodes(
     ]
 
 
+@pytest.mark.asyncio
+async def test_index_nodes_removed_existing(
+    mocker: MockerFixture,
+    repository: Path,
+    session: Session,
+) -> None:
+    """
+    Test ``index_nodes``.
```

Comment:
```suggestion
    Test ``remove_node ``.
```

Reply:
consolidated the test into indexing as it is just a part of that